Zone5/Views/ThirdPartyConnectionsView.swift

```diff
@@ -9,10 +9,18 @@
 import Foundation
 
 public class ThirdPartyConnectionsView: APIView {
+    private class ThirdPartyEndpoint: InternalRequestEndpoint {
+        let uri: String
+
+        init(uri: String) {
+            self.uri = uri
+        }
+    }
 	private enum Endpoints: String, InternalRequestEndpoint {
-		case setThirdPartyConnection = "/rest/users/connections/api/v1/live_activities/set_third_party_token"
-		case hasThirdPartyConnection = "/rest/users/connections/api/v1/live_activities/has_third_party_token"
-		case removeThirdPartyConnection = "/rest/users/connections/api/v1/live_activities/delete_third_party_token"
+        case initializePairing = "/rest/users/connections/pair/"
+		case files = "/rest/files/"
+        case userConnections = "/rest/users/connections"
```

Comment:
indentation off here

Reply:
Fixed, this was a weird indentation, on my local changes was looking ok.